.github/workflows/build_linux_pytorch_wheels.yml

```
@@ -52,9 +53,14 @@ jobs:
           git config --global user.name "therockbot"
           git config --global user.email "therockbot@amd.com"
 
+      - name: Add selected Python version to PATH
+        run: |
+          PYTHON_TAG=cp${{ inputs.python_version }}  # e.g., cp3.12
+          PYTHON_TAG_CLEANED=$(echo "$PYTHON_TAG" | sed 's/\.//g')  # e.g., cp312
+          echo "/opt/python/${PYTHON_TAG_CLEANED}-${PYTHON_TAG_CLEANED}/bin" >> "$GITHUB_PATH"
```

Instead of using sed here, I would rather suggest to adjust the matrix configuration. Instead of having
```
python_version: ["3.11", "3.12"]
```
in `.github/workflows/release_linux_pytorch_wheels.yml` you can set it to
```
python_version: ["cp311-cp311", "cp312-cp312"]
```
and have
```suggestion
          python_dir="/opt/python/${{ inputs.python_version }}"
          if ! [ -x "${python_dir}/bin/python" ]; then
            echo "ERROR: Could not find python: ${python_dir}"
            exit 1
          fi
          echo "/opt/python/${python_dir}/bin" >> "$GITHUB_PATH"
```
here. This also checks if the Python version is available.